app/assets/javascripts/controllers/paper_edit_controller.js.coffee

```diff
@@ -64,19 +64,20 @@ ETahi.PaperEditController = ETahi.BasePaperController.extend
       @get('visualEditor').startEditing()
 
     toggleEditing: ->
-      if @get('lockedBy') #unlocking
+      if @get('lockedBy') #unlocking -> Allowing others to edit
         @set('body', @get('visualEditor.bodyHtml'))
         @set('lockedBy', null)
         @send('stopEditing')
         @get('model').save().then (paper) =>
           @set('saveState', true)
-      else #locking
+      else #locking -> Editing Paper (locking others out)
         @set('lockedBy', @getCurrentUser())
         @get('model').save().then (paper) =>
           @send('startEditing')
           @set('saveState', false)
 
     savePaper: ->
+      return if ETahi.environment == 'test'
```

Comment:
I'm usually wary of these checks on environment in code, why did we have to do this?